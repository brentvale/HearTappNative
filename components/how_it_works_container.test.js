import 'react-native';
import React from 'react';
import HowItWorksContainer from './how_it_works_container';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
	const tree = renderer.create(
		<HowItWorksContainer/>	
	).toJSON();
	expect(tree).toMatchSnapshot();
});
