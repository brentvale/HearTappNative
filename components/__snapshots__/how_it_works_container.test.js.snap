// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
    >
      How It Works
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  />
</View>
`;
